### PR TITLE
fix: update deployed version in footer

### LINKED ISSUE #2982: 🐛 [Bug] Version is incorrect in production
## :bug: Bugreport

We have upgraded to version `v0.3.0`, but on the site it looks like we are still on `v0.2.2`
Screenshot from 2020-02-04 14-29-35

### Steps to reproduce the behavior
1. go  to the root path
2. check the footer for the version number
3. check the image of the app using kubernetes(requires DevOps admin priveleges)
4. ...
5. Profit

### Expected behavior

This should be updated at the time of deployment.

### PR DESCRIPTION
**Pull request by mattwr18**
### Issues

- fixes #2982 

### Todo

- [X] None

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+### [0.3.1](https://github.com/Human-Connection/Human-Connection/compare/v0.3.0...v0.3.1) (2020-02-04)
```
Comment (roschaefer) [inline CHANGELOG.md]: Why this change?

Remove this file from the PR and update the changelog once we merged everything in, I would say?

**Comment by mattwr18** [comment]
closing this as we will update the version in this week's deploy
